lib/notifications.js

```diff
@@ -42,8 +42,9 @@ mbedConnectorApi.prototype.getCallback = function(callback, options) {
                  'callback'),
     headers: {
       accept: 'application/json'
-    }
-  }
+    },
+    timeout: 31000
```

Comment:
TCP Stack in Linux can take up to 120 seconds to terminate the request whenever connection cannot be established. Connector timeout is 30s. so set this to 31s..